README.md

```diff
@@ -1,70 +1,113 @@
+  <a href="#rules">Rules</a> •
+  <a href="#summary">Summary</a> •
+  <a href="#authors">Authors</a> •
+  <a href="#credits">Credits</a>
+</p>
 
-In the project directory, you can run:
+### ✨ [Live Preview]()
 
-### `yarn start`
+## Screenshots
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+<details>
+  <summary>Show Image</summary>
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+<image src="/src/icons/screenshots/"> gif goes here
 
-### `yarn test`
+---
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+Home screen
 
-### `yarn build`
+<image src="/src/icons/screenshots/Home.png">
+   
+---
+   
+Level Select
+   
+<image src="/src/icons/screenshots/LevelSelect.png">
+   
+---
+   
+Battlefield
+   
+<image src="/src/icons/screenshots/Battlefield.png">
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+---
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+</details>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Rules
 
-### `yarn eject`
+   <details>
+  <summary>Show Rules</summary>
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+The goal of the game is to solve the math equation for each side of the battlefield with correct answers earning you a badge.
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+---
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+**How to Play?**
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+<p>The player will use the dinosaur legend on each side of the battlefield to understand the point value of a specific dinosaur.
+We encourage the player to group like dinosaurs together when adding up the totals. Using your imagination with addition or multiplication will make adding their values easier.</p>
+   
+Be careful! The game will add in subtraction dinosaurs as the difficulty increases.
+   
+<p>The player will adjust the input field to equal each teams total point value.
```

Comment:
The same typos from the rules modal are here, too. Just to point them out real fast:

Ln. 62. `each team's total`
Ln. 63. `of your answer, click Battle!`
Ln. 69. `each dinosaur's value`
Ln. 70. `each team's total`
Ln. 71. `If at least one`

Reply:
Thank you! Updated.